pkg/controller/common/helpers.go

```diff
@@ -668,6 +668,35 @@ func MachineConfigFromRawIgnConfig(role, name string, rawIgnCfg []byte) (*mcfgv1
 	}, nil
 }
 
+// KubeletConfigFromIgnConfig creates a KubeletConfig with the provided Ignition config
+func KubeletConfigFromIgnConfig(role, name string, ignCfg interface{}) (*mcfgv1.KubeletConfig, error) {
+	rawIgnCfg, err := json.Marshal(ignCfg)
+	if err != nil {
+		return nil, fmt.Errorf("error marshalling Ignition config: %v", err)
+	}
+	return KubeletConfigFromRawIgnConfig(role, name, rawIgnCfg)
+}
+
+// KubeletConfigFromRawIgnConfig creates a KubeletConfig with the provided raw Ignition config
+func KubeletConfigFromRawIgnConfig(role, name string, rawIgnCfg []byte) (*mcfgv1.KubeletConfig, error) {
+	labels := map[string]string{
+		mcfgv1.KubeletConfigRoleLabelPrefix + role: "",
```

Comment:
Are you labelling the kubeconfig pools.operator.machineconfiguration.openshift.io/master|worker? What happens for a custom pool?

Reply:
Wouldn't a worker kubelet config would update a custom pool since custom pools inherit from workers.

Reply:
Right now, The workerlatencyprofiles are applied only on to the worker pool. Addressed the same in the code.